doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSets/data-sets-caching-con.adoc

```diff
@@ -1,4 +1,4 @@
-[id='data_sets_caching_con_{context}']
+[id='data-sets-caching-con_{context}']
 = Caching
```

Comment:
Suggest to expand for search capabilities:

Something like: "Caching data"

Reply:
Done